examples/AutomatedSummaryReport/main.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/io;
+import ballerinax/slack;
+
+string token = ?;
+
+type ChannelType record {
+    string id;    
+};
+
+type Channels record {|
+    boolean ok;
+    ChannelType[] channels;
+|};
+
+type TextType record{
+    string text;
+};
+
+type History record{
+    boolean ok;
+    TextType[] texts;
+};
+
+
+public function automatedStandUpReport() returns error?{
+    slack:Client cl = check new({
+    auth: {
+        token: value
+    }
+    });
+
+    json response = check cl->/conversations\.list();
+    Channels castedResponse = check response.cloneWithType(Channels);
+
+    string[] latestText; 
+
+    foreach ChannelType channel in castedResponse.channels{
+        json response1 = check cl->/conversations\.history({channel: channel.id});
+        History castedResponse1 = check response1.cloneWithType(History);
+
+        TextType[] texts = castedResponse1.texts;
+        latestText.push(texts[0].text);
```

Comment:
Here should we get the first item or the last item?

Reply:
We need to get the first item as the first item is the latest text in a channel that is sent from the slack server